src/components/views/story-editor-view/story-editor-view.tsx

```diff
@@ -57,6 +61,44 @@ const StoryEditorView: React.FC<Props> = ({
     // eslint-disable-next-line
   }, []);
 
+  /**
+   * Effect that centers the canvas to certain coordinates and selects a knot
+   */
+  React.useEffect(() => {
+    if(!centeredKnot?.id) {
+      return;
+    }
+    
+    const node = engineRef.current.getModel().getNodes().find(item => item.getID() === centeredKnot.id);
```

Comment:
Take engineRef.current.getModel() into own variable and use that instead of using the full path everywhere

Reply:
Took model to a constant of its own